crates/biome_js_analyze/src/semantic_analyzers/a11y/no_positive_tabindex.rs

```diff
@@ -153,12 +157,50 @@ impl Rule for NoPositiveTabindex {
         )
         .note(
             markup!{
-				"Elements with a positive "<Emphasis>"tabIndex"</Emphasis>" override natural page content order. This causes elements without a positive tab index to come last when navigating using a keyboard."
-			}.to_owned(),
+                "Elements with a positive "<Emphasis>"tabIndex"</Emphasis>" override natural page content order. This causes elements without a positive tab index to come last when navigating using a keyboard."
+            }.to_owned(),
+        )
+        .note(
+            markup!{
+                "You are recommended to only use 0 and -1 as "<Emphasis>"tabIndex"</Emphasis>" values. Avoid using "<Emphasis>"tabIndex"</Emphasis>" values greater than 0 and CSS properties that can change the order of focusable HTML elements."
```

Comment:
I think we could avoid the use of `you` by rephrasing as follows:
```suggestion
                "Use only 0 and -1 as "<Emphasis>"tabIndex"</Emphasis>" values. Avoid using "<Emphasis>"tabIndex"</Emphasis>" values greater than 0 and CSS properties that can change the order of focusable HTML elements."
```